include/deal.II/grid/grid_tools.h

```diff
@@ -1239,6 +1239,34 @@ namespace GridTools
 
 
   /**
+   * Given an array of points use the global bounding box description obtained using
+   * GridTools::compute_mesh_predicate_bounding_boxto guess, for each of them,
+   * which process might own it.
+   *
+   * @param[in] global_bboxes : vector of bounding boxes describing the portion of
+   *  mesh with a property for each process.
+   * @param[in] points : the array of points to test.
+   *
+   * @param[out] tuple containing the following parts
+   *  - a vector indicized with ranks of processes. For each rank it contains
+   *   a vector of the indices of points it might own.
+   *  - a map from the index @p unsigned int of the point in @p points
```

Comment:
I might be wrong, but I think that since `unsigned int` is two words, `@p` might not work as expected here. Maybe it needs to be `<code>unsigned int</code>`. Have you by any chance checked this? Same applies in the next line.

Reply:
No I didn't think about this, but I believe you're right!